README.md

```diff
@@ -19,7 +19,7 @@ SPDX-License-Identifier: Apache-2.0
 </h1>
 
 <!--BEGIN_DESCRIPTION-->
-Use this SDK to add real-time video, audio and data features to your Node app. By connecting to a self- or cloud-hosted <a href="https://livekit.io/">LiveKit</a> server, you can quickly build applications like interactive live streaming or video calls with just a few lines of code.
+Use this SDK to add realtime video, audio and data features to your Node app. By connecting to <a href="https://livekit.io/">LiveKit</a> Cloud or a self-hosted server, you can quickly build applications such as multi-modal AI, live streaming, or video calls with just a few lines of code.
```

Comment:
non-hyphenated realtime, but hyphenated multi-modal? seems inconsistent

Reply:
our decision to dehyphenate realtime